### PR TITLE
Create Find_Day_of_a_given_Date.md

### LINKED ISSUE #8438: Find_Day_of_a_given_Date
### Description

Description
Find_Day_of_a_given_Date

### Domain

C/CPP

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by varnikarathee**
<hr>

## Description 📜Find_Day_of_a_given_Date.md

Fixes #8438 

<hr>

## Type of change 📝Find_Day_of_a_given_Date.md

- [X] Documentation (Content Creation in the form of codes or tutorials)
 
<hr>

## Domain of Contribution 📊

- [X] C/CPP

<hr>

## Checklist ✅

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [X] I have performed a self-review of my own code or work.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] My changes generates no new warnings.
- [X] I'm GWOC'21 contributor

<hr>

### CODE REVIEW

**Comment by varnikarathee** [comment]
@vasu-1 Made a new branch varnikarathee-patch1 and made changes. Please merge now.

File: C_CPP/Competitive Programming/Find_Day_of_a_given_Date.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+- F is POSITIVE : Just divide by 7 and see the remainder.  
+- F is NEGATIVE: After dividing by 7, add 7 to remainder which is negative.       
+
+Then find the day as per following table:    
+| SUNDAY |	MONDAY | TUESDAY |	WEDNESDAY |	THURSDAY |	FRIDAY |	SATURDAY |      
+| ------ | ------- | ------- | ---------- | -------- | ------- | --------- |   
+| 0	     | 1       | 2	     | 3	        | 4        | 5       |	6        |   
+
+### IMPLEMENTATION IN CPP:  
+Now we implement the above approach in cpp using OOPS concept:
```
Comment (apeksha235) [inline C_CPP/Competitive Programming/Find_Day_of_a_given_Date.md:28]: add comments to your code

**Review by apeksha235 — CHANGES_REQUESTED** [review]
@varnikarathee

**Comment by varnikarathee** [comment]
@apeksha235 Made the changes as requested. Can u please merge now?

**Comment by varnikarathee** [comment]
@vasu-1can u please merge this . I made all changes

**Comment by Suvraneel** [comment]
@apeksha235  Can you give this a final LGTM (or request changes) before I merge this ?

**Review by apeksha235 — APPROVED** [review]
@Suvraneel yes please merge this

**Pull request merged**